torch/_inductor/codegen/cpp.py

```diff
@@ -1527,6 +1580,7 @@ def load(self, name: str, index: sympy.Expr):
     def store(self, name, index, value, mode=None):
         assert "buf" in name
         var = self.args.output(name)
+        self.cache_fp32_cse_var_before_store(value)
```

Comment:
I guess we only need to invoke this method when `value` is bf16/fp16. Should we add a check before calling it?

Reply:
Added lowp data type check